src/metabase/pulse/render.clj

```diff
@@ -86,6 +86,11 @@
              (= @col-sample-count @row-sample-count 1))
         (chart-type :scalar "result has one row and one column")
 
+        (and (some? maybe-dashcard)
+             (pos? (count (dashboard-card/dashcard->multi-cards maybe-dashcard)))
+             (not (#{:combo} display-type)))
+        (chart-type :multiple "result has multiple card semantics, a multiple chart")
+
```

Comment:
literally just an order bug in `detect-pulse-chart-type`. Glad it turned out to be that simple! And shoutout to Howon for doing the hard parts back in December